settings.gradle.kts

```diff
@@ -80,14 +69,30 @@ gradleEnterprise {
     }
 }
 
+gitRepositories {
+    checkoutsDirectory.set(file(rootProject.projectDir.absolutePath + "/hedera-node/hapi/"))
+    include("hedera-protobufs") {
+        uri.set("https://github.com/hashgraph/hedera-protobufs.git")
+        // choose tag or branch of HAPI you would like to test with
+        // This version needs to match tha HAPI version below in versionCatalogs
+        tag.set("v0.34.0")
+        // do not load project from repo
+        autoInclude.set(false)
+    }
+}
+
 // Define the library catalogs available for projects to make use of
 dependencyResolutionManagement {
     @Suppress("UnstableApiUsage")
     versionCatalogs {
         // The libs of this catalog are the **ONLY** ones that are authorized to be part of the runtime
         // distribution. These libs can be depended on during compilation, or bundled as part of runtime.
         create("libs") {
+            // The HAPI API version to use, this need to match the tag set on gitRepositories above
+            version("hapi-version", "0.34.0")
```

Comment:
since we no have the `hapi` module this name is misleading. Maybe the lang name `hedera-protobuf-java-api-version` is better.

Reply:
Think hapi-version is fine, this is what it has been for ages and I just moved it up. It is also has to match the version of the code in the hapi module which is set with the `tag.set("v0.34.0")`. If they do not match tests will fail. I tried hard to have one variable for both in gradle but failed because of initialization order fighting.